components/admin_console/team_channel_settings/channel/details/channel_profile.tsx

```diff
@@ -64,7 +71,7 @@ export const ChannelProfile: React.SFC<ChannelProfileProps> = (props: ChannelPro
                     </div>
                     <div className='AdminChannelDetails_archiveContainer'>
                         <button
-                            className={'btn btn-secondary ArchiveButton ' + (isArchived ? 'ArchiveButton___archived' : 'ArchiveButton___unarchived')}
+                            className={archiveButtonClassName}
```

Comment:
this would actually be a great place to use the classNames helper instead
```suggestion
                            className={classNames(
                                'btn', 'btn-secondary', 'ArchiveButton', 
                                {'ArchiveButton___archived': isArchived},
                                {'ArchiveButton___unarchived': !isArchived},
                                {'disabled': isDisabled},
                             )}}
```

gonna need an `import classNames from 'classnames';` at the top though and then you can drop lines 40 - 44

Reply:
Good idea. Updated.